lxd/container_exec.go

```diff
@@ -160,6 +160,16 @@ func (s *execWs) Do(op *operation) error {
 
 				if err != nil {
 					shared.LogDebugf("Got error getting next reader %s", err)
+					if er, ok := err.(*websocket.CloseError); ok && er.Code == websocket.CloseAbnormalClosure {
```

Comment:
Can we split that one into a few more lines, would be a bit easier to read I think :)